iterreg/sparse/solvers.py

```diff
@@ -184,7 +182,7 @@ def cd(X, y, alpha, prox=shrink, pen=ell1, max_iter=1_000,
     for t in range(max_iter):
         for j in range(p):
             old = w[j]
-            w[j] = shrink(old + X[:, j].dot(R) / lc[j], alpha / lc[j])
+            w[j] = prox(old + X[:, j].dot(R) / lc[j], alpha / lc[j], lc[j])
```

Comment:
I would pass alpha here, not alpha / lc[j]. It feels less redundant for the soft thresholding for example